hotsite/mainpage/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Convocation</title>
+
+    <link rel="stylesheet" href="style/prototype.css">
+
+    <!--Importing text font-->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700;900&family=Cinzel:wght@400..900&family=MedievalSharp&display=swap" rel="stylesheet">
+
+</head>
+<body>
+
+    <header>
+
+        <img class="Logo" src="src/img/convocation logo.webp" alt="Logo">
```

Comment:
Try to avoid too much break lines is this case. Because it turn your html code to much longer unecesarily and difficult reading.

Example to improve:
```html
<header>
    <img class="Logo" src="src/img/convocation logo.webp" alt="Logo">
    <h1 class="page-title">Welcome to our Cult</h1>
</header>
```